schemas/input/csl-data.json

```diff
@@ -4,425 +4,477 @@
+          }
+        },
+        "editorial-director": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/name-variable"
+          }
+        },
+        "executive-producer": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/name-variable"
+          }
+        },
+        "interviewer": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/name-variable"
+          }
+        },
+        "illustrator": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/name-variable"
+          }
+        },
+        "organizer": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/name-variable"
+          }
+        },
+        "original-author": {
+          "description": "[Deprecated - Use `author` on the `original` related item instead]",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/name-variable"
+          }
+        },
+        "performer": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/name-variable"
+          }
+        },
+        "producer": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/name-variable"
+          }
+        },
+        "recipient": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/name-variable"
+          }
+        },
+        "reviewed-author": {
+          "description": "[Deprecated - Use `author` on the `reviewed` related item instead]",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/name-variable"
+          }
+        },
+        "translator": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/name-variable"
+          }
+        },
+        "accessed": {
+          "$ref": "#/definitions/date-variable"
+        },
+        "available-date": {
+          "$ref": "#/definitions/date-variable"
+        },
+        "container": {
+          "$ref": "#/definitions/date-variable"
+        },
+        "event-date": {
+          "$ref": "#/definitions/date-variable"
+        },
+        "issued": {
+          "$ref": "#/definitions/date-variable"
+        },
+        "original-date": {
+          "description": "[Deprecated - Use a date property on the `original` related item instead]",
+          "$ref": "#/definitions/date-variable"
+        },
+        "submitted": {
+          "$ref": "#/definitions/date-variable"
+        },
+        "abstract": {
+          "type": "string"
+        },
+        "annote": {
+          "type": "string"
+        },
+        "archive": {
+          "type": "string"
+        },
+        "archive_collection": {
+          "type": "string"
+        },
+        "archive_location": {
+          "description": "[Deprecated - Q: what TODO?]",
```

Comment:
Here's the other kind of deprecation annotation; just me asking a question of what, if anything, we should do.

Reply:
I don't think `archive` needs to fit into this related model--it's analogous in structure to `publisher`, rather than to `reviewed`